common/network-common/src/main/java/org/apache/spark/network/protocol/MessageWithHeader.java

```diff
@@ -91,7 +91,7 @@ public long position() {
   }
 
   @Override
-  public long transfered() {
+  public long transferred() {
```

Comment:
This break binary compatibility. Is it OK? @zsxwing

Reply:
It doesn't. The old method is implemented in `AbstractFileRegion.transfered`. In addition, the whole network module is private, we don't need to maintain compatibility.

Reply:
Oh, I see. `AbstractFileRegion.transfered` is `final` so it may break binary compatibility. However, this is fine since it's a private module.

Reply:
I see. Thanks!